app/helpers/application_helper/navbar.rb

```diff
@@ -1,5 +1,33 @@
 module ApplicationHelper
   module Navbar
+    def menu_to_json(position)
```

Comment:
For now, it's okay to have this method here, but after this is done, we should extract it into the `Menu::Manager`. I have an idea about that, just let's not forget it so we need to talk when this is done and we're both in the office.